packages/enzyme-test-suite/package.json

```diff
@@ -49,4 +49,4 @@
     "eslint-plugin-jsx-a11y": "^6.0.3",
     "eslint-plugin-react": "^7.6.1"
   }
-}
```

Comment:
Please make sure all files always have a trailing newline :-)

Reply:
yeah, not sure how this one happened. I'll fix it.